client/resteasy-client/src/test/scala/com/cerner/beadledom/client/resteasy/ExponentialBackoffHttpRequestRetryHandlerSpec.scala

```diff
@@ -1,14 +1,48 @@
 package com.cerner.beadledom.client.resteasy
 
+import java.net.SocketException
+import javax.net.ssl.SSLException
+import org.apache.http.{HttpRequest, HttpVersion}
+import org.apache.http.client.protocol.HttpClientContext
+import org.apache.http.message.BasicRequestLine
+import org.mockito.Mockito
 import org.scalatest.{FunSpec, MustMatchers}
+import org.scalatestplus.mockito.MockitoSugar
```

Comment:
I think this is under a different package with this version of scalatest.

Reply:
At least what I saw with the 3.0.8 release it should be in this one: https://github.com/scalatest/scalatest/blob/release-3.0.8/scalatest/src/main/scala/org/scalatestplus/mockito/MockitoSugar.scala#L50

I'm not sure if the scalatest_{scala.binary.version} artifact id changes things